QueryPartitionedCollection.md

```diff
@@ -0,0 +1,42 @@
+### How to Query Partitioned CosmosDB Collection
+
+With CosmosDB, you can configure [partition key](https://docs.microsoft.com/en-us/azure/cosmos-db/partition-data) for your collection.
+
+Below is an example about how to query partitioned collection with this spring data module.
+
+#### Example 
+
+Given a document entity structure:
+```
+    @Document
+    @Data
+    @AllArgsConstructor
+    public class Address {
+        @Id
+        String postalCode;
+        String street;
+        @PartitionKey
+        String city;
+    }
+```
+
+How to write the repository interface:
+```
+    @Repository
+    public interface AddressRepository extends DocumentDbRepository<Address, String> {
+        void deleteByPostalCodeAndCity(String postalCode, String city);
+        void deleteByCity(String city);   
+        List<Address> findByPostalCodeAndCity(String postalCode, String city);   
```

Comment:
add comments to explain what's the method used for: when query postalcode, need specify 'andcity'